client/entrypoints/amazon-pay-settings/amazon-pay-settings-section.js

```diff
@@ -0,0 +1,203 @@
+import { ADMIN_URL, getSetting } from '@woocommerce/settings';
+import { __ } from '@wordpress/i18n';
+import React, { useMemo } from 'react';
+import {
+	Card,
+	RadioControl,
+	CheckboxControl,
+	Notice,
+} from '@wordpress/components';
+import interpolateComponents from 'interpolate-components';
+import { loadStripe } from '@stripe/stripe-js';
+import styled from '@emotion/styled';
+import ExpressCheckoutPreviewComponent from './express-checkout-button-preview';
+import {
+	useAmazonPayEnabledSettings,
+	useAmazonPayLocations,
+	useAmazonPayButtonSize,
+} from 'wcstripe/data';
+import CardBody from 'wcstripe/settings/card-body';
+import LoadableAccountSection from 'wcstripe/settings/loadable-account-section';
+import { useAccount } from 'wcstripe/data/account/hooks';
+import {
+	useAccountKeysPublishableKey,
+	useAccountKeysTestPublishableKey,
+} from 'wcstripe/data/account-keys/hooks';
+
+const makeButtonSizeText = ( string ) =>
+	interpolateComponents( {
+		mixedString: string,
+		components: {
+			helpText: (
+				<span className="amazon-pay-settings__option-muted-text" />
+			),
+		},
+	} );
+const buttonSizeOptions = [
+	{
+		label: makeButtonSizeText(
+			__(
+				'Small {{helpText}}(40 px){{/helpText}}',
+				'woocommerce-gateway-stripe'
+			)
+		),
+		value: 'small',
+	},
+	{
+		label: makeButtonSizeText(
+			__(
+				'Default {{helpText}}(48 px){{/helpText}}',
+				'woocommerce-gateway-stripe'
+			)
+		),
+		value: 'default',
+	},
+	{
+		label: makeButtonSizeText(
+			__(
+				'Large {{helpText}}(56 px){{/helpText}}',
+				'woocommerce-gateway-stripe'
+			)
+		),
+		value: 'large',
+	},
+];
+
+const AmazonPaySettingsSection = () => {
+	const [ size, setSize ] = useAmazonPayButtonSize();
+	const accountId = useAccount().data?.account?.id;
+	const [ publishableKey ] = useAccountKeysPublishableKey();
+	const [ testPublishableKey ] = useAccountKeysTestPublishableKey();
+
+	const stripePromise = useMemo( () => {
+		return loadStripe(
+			publishableKey || testPublishableKey || 'pk_test_123',
```

Comment:
That's just for testing purposes, right? 🤔

Reply:
It's not for testing. I believe it's used if the Stripe public key is not present yet; we can still see the preview component, just like what we did [here](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/8b8aee85f3447de9fb7f0b923044c8bf55f8a836/client/entrypoints/payment-request-settings/payment-request-settings-section.js/#L141).